dask/dataframe/tests/test_dataframe.py

```diff
@@ -1634,7 +1634,7 @@ def test_combine():
     df1 = pd.DataFrame(
         {
             "A": np.random.choice([1, 2, np.nan], 100),
-            "B": np.random.choice(["a", "b", np.nan], 100),
+            "B": np.random.choice(["a", "b", "nan"], 100),
```

Comment:
This promotion was happening silently before, but `numpy` doesn't tolerate that anymore.